public/index.html

```diff
@@ -18,7 +18,7 @@
       <!-- Use this section to provide information about your relay -->
       <div class="panel panel-primary info-pane" ng-hide="infoVisible">
         <div class="panel-heading">
-          <h3 class="panel-title">INFO: This is a <i>'Zax Test'</i> instance at <b><span id="hostblock">https://zax_test.vault12.com</span></b>
+          <h3 class="panel-title">INFO: This is a <i>Zax</i> instance at <b><span id="hostblock"></span></b>
```

Comment:
I think this is missing the info for the server url - don't you want https://z.vault12.com to be here?

Reply:
That is correct: client script writes in `#hostblock` url index is served from